resources/styles/organisms/site-footer/_index.scss

```diff
@@ -15,32 +15,57 @@
         align-items: start;
     }
 
-    &__links {
-        grid-column: 5 / 11;
-        justify-self: center;
+
+    &__identity {
+        display: grid;
+        grid-column: main;
+        row-gap: config.$thin-h-space;
+
+        @include at(lg) {
+            grid-column: 2 / 5;
+        }
     }
 
     &__brand {
-        grid-column: 2 / 5;
         align-items: center;
-        color: config.$primary;
+        color: config.$yellow-100;
         display: flex;
         gap: config.$thin-h-space;
         padding: config.$thin-v-space 0;
     }
 
     &__socialLinks {
-        grid-column: 11 / 14;
-        justify-self: end;
         display: flex;
         gap: config.$thin-h-space;
     }
 
-    &__utility {
+    &__links {
+        display: grid;
         grid-column: main;
+        row-gap: config.$thin-h-space;
+
+        @include at(lg) {
+            column-gap: config.$h-space;
+            display: grid;
+            grid-column: 5 / 14;
+            grid-template-columns: repeat(5, minmax(0, 1fr));
```

Comment:
I'd love to see us use `grid-auto-flow: column` or `auto-fit/auto-fill` here instead of hard coding the number of columns, depending on what you think the sensible behavior for smaller screen widths/smaller numbers of columns.

Reply:
i tried some auto-fit stuff